libopendmarc/opendmarc_policy.c

```diff
@@ -481,6 +485,11 @@ opendmarc_policy_store_dkim(DMARC_POLICY_T *pctx, u_char *d_equal_domain, int dk
 			(void) free(pctx->dkim_domain);
 			pctx->dkim_domain = NULL;
 		}
+		if (pctx->dkim_selector != NULL)
```

Comment:
We should probably turn this into a macro since it's repeated several times.  Doesn't need to be done here, more a note for later.